decrypt_test.go

```diff
@@ -202,7 +202,7 @@ yoP7LBNaX/r83/SsGGbtW618sar+27wWRlGeZvFq8hwh+qyj3S7wtY7wx5weR7eZ
 S9K/MB9npTbIjDUidByTIhpk0hS55s1jF6U0wlmV0oUmC4wBIebyo1cL/CMPRv+a
 z/D9/ZvUoaotiATRWSKLCmxapcmGbMxhHHYWYWtzPCt1
 -----END CERTIFICATE-----
------BEGIN RSA PRIVATE KEY-----
+-----BEGIN PRIVATE KEY-----
```

Comment:
This should be `RSA PRIVATE KEY` as it is encoded using PKCS #1. I haven't checked the rest, but I assume they are also PKCS #1.